index.js

```diff
@@ -1 +1,90 @@
-// implement your API here
+const express = require('express');
+
+const db = require('./data/db.js');
+
+const server = express();
+
+const { users } = db;
```

Comment:
I probably would not use this as db is not exporting anything called users that I am aware of and I don't see anywhere that you are using this. It could cause an issue in your code down the line as it's a global variable that could find its way into one of your other code blocks by mistake.